composer/spec/fixtures/packagist_response.json

```diff
@@ -1,2726 +1,2232 @@
                     "php-console/php-console": "^3.1.3",
-                    "phpunit/phpunit": "~4.5",
                     "phpunit/phpunit-mock-objects": "2.3.0",
-                    "ruflin/elastica": ">=0.90 <3.0",
-                    "sentry/sentry": "^0.13",
-                    "swiftmailer/swiftmailer": "~5.3"
+                    "jakub-onderka/php-parallel-lint": "0.9"
                 },
+                "support": {
+                    "issues": "https://github.com/Seldaek/monolog/issues",
+                    "source": "https://github.com/Seldaek/monolog/tree/1.22.1"
+                }
+            },
+            {
+                "version": "1.22.0",
+                "version_normalized": "1.22.0.0",
                 "source": {
-                    "reference": "bad29cb8d18ab0315e6c477751418a82c850d558",
+                    "url": "https://github.com/Seldaek/monolog.git",
                     "type": "git",
-                    "url": "https://github.com/Seldaek/monolog.git"
+                    "reference": "bad29cb8d18ab0315e6c477751418a82c850d558"
                 },
-                "suggest": {
-                    "aws/aws-sdk-php": "Allow sending log messages to AWS services like DynamoDB",
-                    "doctrine/couchdb": "Allow sending log messages to a CouchDB server",
-                    "ext-amqp": "Allow sending log messages to an AMQP server (1.0+ required)",
-                    "ext-mongo": "Allow sending log messages to a MongoDB server",
-                    "graylog2/gelf-php": "Allow sending log messages to a GrayLog2 server",
-                    "mongodb/mongodb": "Allow sending log messages to a MongoDB server via PHP Driver",
-                    "php-amqplib/php-amqplib": "Allow sending log messages to an AMQP server using php-amqplib",
-                    "php-console/php-console": "Allow sending log messages to Google Chrome",
-                    "rollbar/rollbar": "Allow sending log messages to Rollbar",
-                    "ruflin/elastica": "Allow sending log messages to an Elastic Search server",
-                    "sentry/sentry": "Allow sending log messages to a Sentry server"
+                "dist": {
+                    "url": "https://api.github.com/repos/Seldaek/monolog/zipball/bad29cb8d18ab0315e6c477751418a82c850d558",
+                    "type": "zip",
+                    "shasum": "",
+                    "reference": "bad29cb8d18ab0315e6c477751418a82c850d558"
                 },
                 "time": "2016-11-26T00:15:39+00:00",
-                "type": "library",
-                "uid": 1099516,
-                "version": "1.22.0",
-                "version_normalized": "1.22.0.0"
+                "support": {
+                    "issues": "https://github.com/Seldaek/monolog/issues",
+                    "source": "https://github.com/Seldaek/monolog/tree/1.22.0"
+                }
             },
-            "v1.22.1": {
```

Comment:
note to self: this was a customization we added to ensure we parse `"v*"` versions correctly, so need to put it back... 

check for other customizations that need to be added here:
* https://github.com/dependabot/dependabot-core/commits/main/composer/spec/fixtures/packagist_response.json
* https://github.com/dependabot/dependabot-core/commits/a2c111565b61e371289ca0b90d7a3682ff2590c1/spec/fixtures/php/packagist_response.json?browsing_rename_history=true&new_path=composer/spec/fixtures/packagist_response.json&original_branch=main

Reply:
These were handled in:
* https://github.com/dependabot/dependabot-core/pull/6332
* https://github.com/dependabot/dependabot-core/pull/6335